components/UnorderedList/src/index.scss

```diff
@@ -22,3 +22,28 @@
 .denhaag-unordered-list--lower-alpha {
   list-style-type: var(--denhaag-unordered-list-lower-alpha-list-style-type);
 }
+
+.denhaag-unordered-list ul {
+  @extend .denhaag-unordered-list;
+
+  margin-inline-start: initial;
+  margin-inline-end: initial;
+  margin-block-start: initial;
+  margin-block-end: initial;
+}
+
+.denhaag-unordered-list li {
+  line-height: var(--denhaag-unordered-list-line-height, 1.5);
+}
+
+.denhaag-unordered-list li > p {
```

Comment:
I would very much avoid directly selecting html elements instead of BEM class names. This behaviour might be consistently needed for the website but will cause unexpected behaviour for others. Please use `utrecht-unordered-list__item` or `denhaag-unordered-list__item` and `utrecht-paragraph` instead.

Reply:
I removed the fallback css now